src/map.h

```diff
@@ -609,7 +609,8 @@ class map
 // Flags: 3D
     std::string features( const tripoint &p ); // Words relevant to terrain (sharp, etc)
     bool has_flag( const std::string &flag, const tripoint &p ) const;  // checks terrain, furniture and vehicles
-    bool can_put_items( const tripoint &p ); // True if items can be placed in this tile
+    bool can_drop_items( const tripoint &p ) const; // True if items can be dropped in this tile
+    bool can_put_items( const tripoint &p ) const; // True if items can be placed in this tile
```

Comment:
ter_fern? Can you explain what this means?

I'm rather new to the game and not yet that familiar with the game concepts. I don't understand the use case distinction between the two methods. From the end user's perspective, when would he be able put items, and when would he be able to put items _ter fern_?